src/operator/toPromise.ts

```diff
@@ -7,8 +7,51 @@ export function toPromise<T>(this: Observable<T>, PromiseCtor: typeof Promise):
 /* tslint:disable:max-line-length */
 
 /**
- * @param PromiseCtor
- * @return {Promise<T>}
+ * Converts an Observable sequence to a ES2015 compliant promise.
+ *
+ * @example
+ * // Using normal ES2015
+ * let source = Rx.Observable
+ *   .just(42)
+ *   .toPromise();
+ *
+ * source.then((value) => console.log('Value: %s', value));
+ * // => Value: 42
+ *
+ * // Rejected Promise
+ * // Using normal ES2015
+ * let source = Rx.Observable
+ *   .throw(new Error('woops'))
+ *   .toPromise();
+ *
+ * source
+ *   .then((value) => console.log('Value: %s', value))
+ *   .catch((err) => console.log('Error: %s', err));
+ * // => Error: Error: woops
+ *
+ * // Setting via the config
+ * Rx.config.Promise = RSVP.Promise;
+ *
+ * let source = Rx.Observable
+ *   .just(42)
```

Comment:
Ah right. The [examples](https://github.com/Reactive-Extensions/RxJS/blob/master/doc/api/core/operators/topromise.md) from the RxJS 4 docs were pretty thorough so I copy/pasted them and forgot to fix this. Fixed in https://github.com/ReactiveX/rxjs/pull/2208/commits/aede7387134f4d1b9cd83f603726775f06475e11.